medusa/restore_node.py

```diff
@@ -179,6 +179,7 @@ def invoke_sstableloader(config, download_dir, keep_auth, fqtns_to_restore, stor
                                           "github:apache/", "githubCOLONapacheSLASH"),
                                           '-d', hostname_resolver.resolve_fqdn() if cassandra_is_ccm == 0
                                           else '127.0.0.1',
+                                          '--conf-path', config.cassandra.config_file,
```

Comment:
@adejanovski shouldn't this change be made also in the case we are not using ccm?

This is enough to fix the sstable loader with 4.0? wow :)

Reply:
This code path is used for all cases, ccm and plain old Cassandra cluster, unless I'm missing something.

Reply:
Oh yes indeed, you're right. I have been confused by the previous line. My bad 👍 .